frida_tools/repl.py

```diff
@@ -3,7 +3,7 @@
 
 import signal
 import threading
-
+import os
```

Comment:
Let's move this up so they are in alphabetic order. Also we need to keep two blank lines between the imports and `main`.